zerver/tests/test_invite.py

```diff
@@ -108,7 +108,7 @@ def test_query_counts_for_new_user_using_default_streams(self) -> None:
 
         new_user = self.create_simple_new_user(realm, "alice@zulip.com")
 
-        with self.assert_database_query_count(11):
+        with self.assert_database_query_count(12):
```

Comment:
Here also query count increases just because of accessing `realm.can_access_all_users_group.name`.